### PR TITLE
Add IQMFakeApollo

### PR DESCRIPTION
**Pull request by JMuff22**
Hi, this pull request adds `IQMFakeApollo` to provide noisy simulation of IQM's Apollo chip. I created the changes based off `IQMFakeAdonis`. The values are currently placeholder values which are open to be changed by the maintainers. 

Should any documentation be added regarding this PR? Let me know and I will add it.

### CODE REVIEW

**Comment by q-mat-beu** [comment]
Hi Jake, sorry for the late response and thanks for the contribution. MR looks good to me I will try and get the QPU specs. In the mean time can you merge latest changes in `main` please?

**Comment by JMuff22** [comment]
> Hi Jake, sorry for the late response and thanks for the contribution. MR looks good to me I will try and get the QPU specs. In the mean time can you merge latest changes in `main` please?

No worries. I merged the latest changes (initially forgot to sync my fork) and fixed the tests and I believe you added some additional parameters to the fake backends and tests for them. Tests run fine locally.

**Comment by JMuff22** [comment]
> Hi @JMuff22 ! Please see this PR: [JMuff22#1](https://github.com/JMuff22/qiskit-on-iqm/pull/1)

Thanks @olliahonen ! Merged + added changelog entry.

File: src/iqm/qiskit_iqm/fake_backends/fake_apollo.py [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+            ["QB18", "QB14"],
+            ["QB16", "QB15"],
+            ["QB19", "QB15"],
+            ["QB16", "QB17"],
+            ["QB16", "QB20"],
+            ["QB18", "QB19"],
+            ["QB19", "QB20"],
+        ],
+    )
+    error_profile = IQMErrorProfile(
```
Comment (q-mat-beu) [inline src/iqm/qiskit_iqm/fake_backends/fake_apollo.py:80]:
```suggestion
     # Note that these specs are ballpark numbers and don't correspond directly to a specific device
    error_profile = IQMErrorProfile(
```

**Comment by q-mat-beu** [comment]
Left one more comment and master has to be merged then I will approve.

**Review by q-mat-beu — APPROVED** [review]

**Comment by q-mat-beu** [comment]
The pipelines say you still need to run black then we can merge.

**Comment by JMuff22** [comment]
The additional line added caused the formatting to fail :) . Should be fixed now.

**Comment by q-mat-beu** [comment]
Sorry about that 😁 Please go ahead and squash and merge.

**Comment by JMuff22** [comment]
I think we need a maintainer to squash and merge @olliahonen

**Pull request merged**